### PR TITLE
add gworkspace records for podcast

### PR DESCRIPTION
**Pull request by tregsthedev**

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1506,9 +1506,27 @@ plis:
...
-  ttl: 1
+  - ttl: 1
   type: CNAME
   value: cname.vercel-dns.com.
+- ttl: 3600
+  type: TXT
+  values: 
+  - google-site-verification=qA3aMUHA0TlUHLAITzYM7-qVal8iUnvramYq2msLc8U
+- ttl: 3600
+type: MX
```
Comment (reesericci) [inline hackclub.com.yaml:1517]: this is indented wrong

Comment (reesericci) [reply hackclub.com.yaml:1517]: @tregsthedev please fix

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1506,9 +1506,26 @@ plis:
...
+      - exchange: aspmx.l.google.com.
+        preference: 1
+      - exchange: alt1.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt2.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt3.aspmx.l.google.com.
+        preference: 10
+      - exchange: alt4.aspmx.l.google.com.
+        preference: 10
```
Comment (cjdenio) [inline hackclub.com.yaml:1528]:
```suggestion
  - ttl: 3600
    type: TXT
    values: 
    - google-site-verification=qA3aMUHA0TlUHLAITzYM7-qVal8iUnvramYq2msLc8U
  - ttl: 3600
    type: MX
     values:
        - exchange: aspmx.l.google.com.
          preference: 1
        - exchange: alt1.aspmx.l.google.com.
          preference: 5
        - exchange: alt2.aspmx.l.google.com.
          preference: 5
        - exchange: alt3.aspmx.l.google.com.
          preference: 10
        - exchange: alt4.aspmx.l.google.com.
          preference: 10
```

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1506,9 +1506,26 @@ plis:
   type: CNAME
   value: plis-hackclub.netlify.app.
 podcast:
-  ttl: 1
+  - ttl: 1
   type: CNAME
```
Comment (sampoder) [inline hackclub.com.yaml:1510]:
```suggestion
  type: ALIAS
```

**Comment by maxwofford** [comment]
I haven't seen any activity so i'm closing for now. @tregsthedev if you need help with this let me know!

**Pull request closed without merging**